lsp/src/test/scala/org/ensime/lsp/rpc/JsonRpcMessageSpec.scala

```diff
@@ -434,22 +377,6 @@ class JsonRpcMessageSpec extends FreeSpec {
                    |}""".stripMargin.parseJson
       willFailToDecode[JsonRpcResponseMessage, DeserializationException](json)
     }
-    "without an error or a result" - {
```

Comment:
why delete this?

Reply:
that might be an accident.

Reply:
decodes as `JsonRpcResponseSuccessMessage(2.0,JsNull,NumberId(0))` which is valid.